sentences/ms/_common.yaml

```diff
@@ -1,12 +1,60 @@
 language: ms
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: "Maaf, saya tidak faham arahan anda"
+    no_area: "Tiada kawasan {{ area }} dijumpai"
+    no_domain: "{{ domain }} tiada di kawasan {{ area }}"
+    no_device_class: "{{ device_class }} tiada di kawasan {{ area }}"
+    no_entity: "Peranti atau entiti {{ entity }} tidak ditemui"
+    handle_error: "Maaf, berlaku ralat semasa intent sedang dijalankan"
+lists:
+  color:
+    values:
+      - in: "putih"
+        out: "white"
+      - in: "hitam"
+        out: "black"
+      - in: "merah"
+        out: "red"
+      - in: "oren"
+        out: "orange"
+      - in: "kuning"
+        out: "yellow"
+      - in: "hijau"
+        out: "green"
+      - in: "biru"
+        out: "blue"
+      - in: "ungu"
+        out: "purple"
+      - in: "perang"
+        out: "brown"
+  brightness:
+    range:
+      type: "peratus"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "suhu"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "celsius"
+      - "fahrenheit"
+      - in: "f"
+        out: "fahrenheit"
+expansion_rules:
+  name: "[the] {name}"
+  area: "[the] {area}"
+  what_is: "(apa itu | apakah | apa)"
```

Comment:
This one does not wrap a slot name like `{name}` and so the key `what_is` should be updated to be in Malay.

Reply:
(this applies to all expansion rules that don't wrap slots)

Reply:
noted.. removing this for time being. thank you